pkg/webui/console/store/actions/gateways.js

```diff
@@ -48,6 +48,16 @@ import {
 
 export const SHARED_NAME = GATEWAY
 
+export const CREATE_GTW_BASE = 'CREATE_GTW'
+export const [
+  { request: CREATE_GTW, success: CREATE_GTW_SUCCESS, failure: CREATE_GTW_FAILURE },
+  { request: createGateway, success: createGatewaySuccess, failure: createGatewayFailure },
+] = createRequestActions(CREATE_GTW_BASE, (ownerId, gateway, isAdmin) => ({
```

Comment:
```suggestion
] = createRequestActions(CREATE_GTW_BASE, (ownerId, gateway, isUserOwner) => ({
```
Where is that `isAdmin` coming from?